### ISSUE TITLE
Make api accessible via http

### ISSUE THREAD

**Issue by simonschaufi** [portachtzig/neos-piwik#3]
Thank you for providing this module!

I don't get the point why https is required. Many people host piwik on shared webhosts that don't have https support easily. please provide a configuration setting for protocol as well.

**Comment by sarahRehbock** [comment]
Hi Simon!

The idea of the https enforcement is, that we don't want the superuser's token (which is needed to fetch the list of Piwik sites) to be an unencrypted URL parameter,as you can see in the following API call that fetches all visits for the current page:

```
https://your.domain.com/index.php?module=API&format=json&method=Actions.getPageUrl&date=last1&period=year&pageUrl=http%3A%2F%2Fpiwiktest.testbox.lan%2F&idSite=4&token_auth=#your_token_as_url_param#'
```

You can easily change this for your own use by changing "https" to "http" in the `callAPI` and `getNodeStatistics` methods in `Service/DataSource/Reporting.php` class.

We'll think about making this configurable in the settings. In the meantime you have to "force" http as mentioned above.

For the moment we just leave this issue open, so maybe other's can leave their comments on this problem.

Thanks for your feedback and for using this module!

Best,
Sarah

**Comment by sarahRehbock** [comment]
closed by latest version [1.0.4](https://github.com/portachtzig/neos-piwik/tree/1.0.4) which introduces the possibility to choose the preferred protocol in the backend module.
